components/resource-creation/ResourceDisplay.tsx

```diff
@@ -84,6 +87,63 @@ function ResourceDisplay() {
     setSelectedDataRequirement({ name: '', content: null });
   };
 
+  // Return type is a an string array, the first element is the formatted date and the second is the resource type.
```

Comment:
```suggestion
  // Return type is a string array, the first element is the formatted date and the second is the resource type.
```
But I would change this type anyways (see below comment)